proto/streamlit/proto/Slider.proto

```diff
@@ -21,7 +21,7 @@ message Slider {
   string label = 2;
   string format = 7;
 
-  repeated double default = 8;
+  repeated double value = 8;
```

Comment:
Like Button.proto comment: To change the protobuf, choose a new numerical value for "value" and explicitly deprecate the word "default" as well as its number.